src/coreclr/src/dlls/mscordac/CMakeLists.txt

```diff
@@ -31,11 +31,11 @@ else(CLR_CMAKE_HOST_WIN32)
     # Add dependency on export file
     add_custom_target(mscordaccore_exports DEPENDS ${EXPORTS_FILE})
 
-    if(CLR_CMAKE_HOST_OSX OR CLR_CMAKE_HOST_FREEBSD OR CLR_CMAKE_HOST_NETBSD)
+    if(CLR_CMAKE_HOST_OSX OR CLR_CMAKE_HOST_NETBSD OR CLR_CMAKE_TARGET_FREEBSD)
         generate_exports_file(${DEF_SOURCES} ${EXPORTS_FILE})
-    endif(CLR_CMAKE_HOST_OSX OR CLR_CMAKE_HOST_FREEBSD OR CLR_CMAKE_HOST_NETBSD)
+    endif(CLR_CMAKE_HOST_OSX OR CLR_CMAKE_HOST_NETBSD OR CLR_CMAKE_TARGET_FREEBSD)
 
-    if(CLR_CMAKE_HOST_LINUX)
+    if(CLR_CMAKE_HOST_LINUX AND NOT CLR_CMAKE_TARGET_FREEBSD)
```

Comment:
Why is this excluded for FreeBSD?

Reply:
it did not build for me and it seems like we were not building it for CLR_CMAKE_HOST_FREEBSD. At this point the goal is to make cross-compiled build same as original build on  FreeBSD